### ISSUE TITLE
Unclear spec error message as output

### ISSUE THREAD

**Issue by max-codeware** [crystal-lang/crystal#9205]
I am trying to fix the parser spec for a modification I made, but I get this output as error
```
Parser parses "b = 0;puts foo -1, b"
     Failure/Error: node.should eq(Expressions.from(local_expected_node))

       Expected: b = 0
       puts(foo(-1, b))
        : Crystal::Expressions
            got: b = 0
       puts(foo(-1, b))
        : Crystal::Expressions

     # spec/compiler/parser/parser_spec.cr:21
```

I don't know if there is any other issue open like this, as I don't know how to look for it, and I also guess it's just a matter of how the ast node is constructed, but what can I do to know what's wrong?

**Comment by asterite** [comment]
Delete these three lines:

https://github.com/crystal-lang/crystal/blob/e2717c6cb1f639cd3f89a0889ec7c2d164ed5b7f/src/compiler/crystal/syntax/to_s.cr#L6-L8

Also feel free to send a PR with those three (four) lines removed.

**Comment by straight-shoota** [comment]
I agree it would be more reasonable to let `inspect` print a structured view of the AST node instead of stringified code.

**Comment by max-codeware** [comment]
It would be nicer to have a more readable format for spec. Whole `inspect` is indeed more meaningful, but in case of articulated AST nodes it becomes hard to read.
By the way, I'll send the PR for the moment
